app/js/populateCategoryDropdown.ts

```diff
@@ -20,4 +20,13 @@ function populateNewUserCategoryDropdown () {
             document.querySelector<HTMLElement>('#category_id').innerHTML = template(testsObject);
         });
     })
+}
+
+function changeNewUserCategoryDropdown() {
```

Comment:
Could you add a comment for this function like the others in the system?

Would it be possible so that the dropdown defaults to the currently set value? Unsure how easy it would be to do this